cms/static/js/views/xblock_outline.js

```diff
@@ -44,6 +44,19 @@ define(["jquery", "underscore", "gettext", "js/views/baseview", "js/views/utils/
                 this.renderTemplate();
                 this.addButtonActions(this.$el);
                 this.addNameEditor();
+
+                if (this.model.get('is_visible') != null) {
```

Comment:
Add test coverage of this.

Subsections are expanded by default, so I'm a little surprised that the locators are not already present. Or perhaps they are, but you are being complete in case is_visible is ever specified on a Section? I can support that, but I'm curious about how this interacts with the default expanded state for subsections.